src/utils/arkhamdb.ts

```diff
@@ -2,11 +2,36 @@ import type { ResolvedDeck } from "@/store/lib/types";
 
 export function redirectArkhamDBLinks(evt: React.MouseEvent) {
   evt.preventDefault();
-  if (evt.target instanceof HTMLAnchorElement) {
-    const href = evt.target.getAttribute("href");
+  const target = evt.target as HTMLElement;
+
+  /**
+   * If one clicked on bolded anchor or image anchor, need to look for `<a>` on its parent.
+   */
+  function searchForAnchorParentRecursive(
```

Comment:
Why do we need this function? Shouldn't the click event bubble up the DOM and eventually register on a parent `a` element? The only case where that would not happen is if a child calls `.stopPropagation()` on it.

Reply:
I think we would have to check `evt.currentTarget` in that case instead of `evt.target`, then it should work without the manual check. 🤔

Reply:
@5argon Oh right, that could be. In that case ignore, we can merge as is